client/dashboard/dashboard-charts/index.js

```diff
@@ -109,14 +119,56 @@ class DashboardCharts extends Component {
 		);
 	}
 
+	renderIntervalSelector() {
+		const allowedIntervals = getAllowedIntervalsForQuery( this.props.query );
+		if ( ! allowedIntervals || allowedIntervals.length < 1 ) {
```

Comment:
Should we render the interval selector if there's only one option?

Reply:
Good question. 🤔   I know we still continue to show it in other charts, so maybe we should remove it across all charts if there is only 1 interval option.  However, it may just act as an information cue to users, but not entirely sure.  @LevinMedia can you weigh in here?

Reply:
@jeffstieler @joshuatf For consistency's sake I think we should. Thanks!

Reply:
I am noticing a strange behavior when I refresh the page - all of the available chart are appearing for a spit second. Some finish loading, others don't then they disappear. 

<img width="2034" alt="screen shot 2019-01-07 at 9 10 49 pm" src="https://user-images.githubusercontent.com/4500952/50810920-e6fef480-12c0-11e9-992a-43b598e44306.png">